tests/config/checkCoverage.js

```diff
@@ -51,6 +51,11 @@ if (browserName !== 'chromium') {
 if (browserName === 'webkit')
   api.delete('browserContext.clearPermissions');
 
+// WebKit on Linux has no sizes caused by libsoup, waiting for libsoupv3
+if (browserName === 'webkit') {
+  api.delete('request.sizes');
```

Comment:
I think we have body / transfer sizes through dataReceived in WebKit, don't we?

Reply:
Not on Linux unfortunately, there it's blocked by libsoup3.

Reply:
So no data size in dataReceived?